src/Web.Api/Features/Github/GithubController.cs

```diff
@@ -72,4 +73,22 @@ await _serviceProvider.GetRequiredService<DeleteGithubProcessingJobHandler>()
             return NotFound(ex.Message);
         }
     }
+
+    [HttpDelete("profiles/{username}")]
+    public async Task<IActionResult> DeleteGithubProfile(
+        string username,
+        CancellationToken cancellationToken)
+    {
+        try
```

Comment:
Avoid using try catch there. The NotFound Exception will be thrown directly in the handler